### PR TITLE
Add option to rotate images

### PR DESCRIPTION
**Pull request by YaoiFangirl420**
cf. https://github.com/vector-im/riot-web/issues/9257

Add rotate clockwise/counterclockwise buttons to <ImageView>

### CODE REVIEW

**Comment by YaoiFangirl420** [comment]
Looks like:
rotate-screenshot

I don't really care if the rotate buttons are in the corner there or someplace else, it was just a convenient spot to put them.

**Comment by jryans** [comment]
Thanks for the contribution! 😁 Please update the PR or each commit with your sign off as described in [CONTRIBUTING.rst](https://github.com/matrix-org/matrix-js-sdk/blob/develop/CONTRIBUTING.rst).

File: src/components/views/elements/ImageView.js [hunk 1]
```diff
@@ -83,17 +86,29 @@ module.exports = React.createClass({
                 }).done();
             },
         });
-    },
+    };
 
-    getName: function() {
+    getName = () => {
```
Comment (turt2live) [inline src/components/views/elements/ImageView.js]: not sure this needs to be bound like this

File: src/components/views/elements/ImageView.js [hunk 2]
```diff
@@ -168,15 +184,26 @@ module.exports = React.createClass({
...
                 <div className="mx_ImageView_lhs">
                 </div>
                 <div className="mx_ImageView_content">
-                    <img src={this.props.src} title={this.props.name} style={style} />
+                    <img src={this.props.src} title={this.props.name} style={effectiveStyle} className="mainImage" />
                     <div className="mx_ImageView_labelWrapper">
                         <div className="mx_ImageView_label">
-                            <AccessibleButton className="mx_ImageView_cancel" onClick={ this.props.onFinished }><img src={require("../../../../res/img/cancel-white.svg")} width="18" height="18" alt={ _t('Close') } /></AccessibleButton>
+                            <AccessibleButton className="mx_ImageView_rotateCounterClockwise" onClick={ this.rotateCounterClockwise }>
+                                <img src={require("../../../../res/img/rotate-ccw.svg")} alt={ 'Rotate counter-clockwise' } width="18" height="18" />
```
Comment (turt2live) [inline src/components/views/elements/ImageView.js]: The alt needs to be translatable: `_t('Rotate counter-clockwise')` and `yarn i18n`

Applies to the other image as well.

**Pull request merged**